RSVPLink.sql

```diff
@@ -0,0 +1,47 @@
+USE [WeddingSiteBuilder]
+GO
+
+/****** Object:  Table [dbo].[RSVPLink]    Script Date: 12/3/2015 10:09:29 PM ******/
+SET ANSI_NULLS ON
+GO
+
+SET QUOTED_IDENTIFIER ON
+GO
+
+SET ANSI_PADDING ON
+GO
+
+CREATE TABLE [dbo].[RSVPLink](
+	[RSVPLinkID] [bigint] IDENTITY(1,1) NOT NULL,
+	[WeddingID] [bigint] NOT NULL,
+	[PersonID] [bigint] NOT NULL,
```

Comment:
Also instead of person Id we may want to make this an AttendeeId then we can get rid of the weddingId